### PR TITLE
Added a updated Readme File 

### PR DESCRIPTION
**Pull request by ShivamAgarwal-code**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   This PR include the Readme.md update
- **Why was this change needed?** (You can also link to an open issue here)
These changes should be implemented to improve the integrity of the README
- **Other information**:
#1688 
The changes looking good than the existing ones.
It follows contributing Guidelines and Code of Conduct

Please review and merge
Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,39 +92,38 @@ import { Novu } from '@novu/node';
...
+      logo: 'https://happycorp.com/logo.png',
+    },
+  },
+});
 ```
 
 ## Embeddable Notification Center
 
-Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
+Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iFrame embed if you are not using React.
```
Comment (p-fernandez) [inline README.md:115]:
```suggestion
Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
```
Arguably this is an opinionated capitalisation that is not used by the W3C.
https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html

File: README.md [hunk 2]
```diff
@@ -45,35 +45,34 @@ The ultimate service for managing multi-channel notifications with a single API.
...
 ## ⭐️ Why Novu?
 
-Building a notification system is hard, at first, it seems like just sending an email, but in reality, it's just the beginning. Users today expect a multi-channel communication experience via email, SMS, push, chat, and other channels. An ever-growing list of providers is popping up daily, and notifications are spread around the code. Novu's goal is to simplify notifications and provide developers with the tools to create meaningful communication between the system and its users.
+Building a notification system is hard, at first, it seems like just sending an email, but in reality, it's just the beginning. Users today expect a multi-channel communication experience via email, SMS, push, chat, and other channels. An ever-growing list of providers is popping up daily, and notifications are spread around the code. Novu's goal is to simplify notifications and provide developers with the tools to create meaningful communication between a system and its users.
 
 ## ✨ Features
 
 - 🌈 Single API for all messaging providers (Email, SMS, Push, Chat)
-- 💅 Easily manage notifications over multiple channels
+- 💅 Easily manage notification over multiple channels
```
Comment (p-fernandez) [inline README.md:53]: Existing copy is accurate. Please roll this change back.

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please address the changes requested. 🙏🏻

**Comment by scopsy** [comment]
Closing this for now due to inactivity.

**Pull request closed without merging**